### ISSUE TITLE
Upload & Convert Functionality

### ISSUE THREAD

**Issue by robredpath** [OpenDataServices/cove#602]

**Comment by Bjwebb** [comment]
I'm working on a flatten-tool pull request for the functionality in flatten-tool https://github.com/OpenDataServices/flatten-tool/pull/142
There are some points there that I think need to be done before the code is merged into master, these are what I'm currently working on.

Additional issues with the conversion, that don't necessarily need doing this sprint (but we may want to):
* We have to write /0/ which is unnatural for XML, I suspect we can avoid this by treating everything as a list
* No @version on iati-activities. This should be fixed by @kindly's meta-tab work https://github.com/OpenDataServices/flatten-tool/compare/master...meta-tab
* No support XML namespaces (how extensions are handled in IATI)
* No support for human readable titles (either a mapping or extra rows https://github.com/OpenDataServices/flatten-tool/issues/130)

**Comment by robredpath** [comment]
Talked about this today. To make our shared understanding explicit, this is making a website that I can go to, upload a spreadsheet based on a certain template, and get a page back with a download link to download some XML
